src/yaml-file.ts

```diff
@@ -34,6 +34,16 @@ export class YamlFile {
    */
   private readonly patchOperations: JsonPatch[];
 
+  /**
+   * Internal storage for commentBefore accessors.
+   */
+  private _commentBefore?: string;
```

Comment:
```suggestion
  private commentBefore?: string;
```

Since these are private, I see no reason to underscore

Reply:
The underscore is for disambiguating from the getter/setter.

The getter and setter were added out of habit for future expansion where there may be a desire to possibly alter the way that these values are stored, add hooks on when the values is set or read, etc.

These are unlikely in this case I suppose.